<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Acc</class>
 <widget class="QDialog" name="Acc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>561</width>
    <height>346</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>A-Parking</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);
                          border-style: inset;
                          border-style: outset;
                          border-width: 1px;
                           border: 2px solid #000000;
                          border-radius: 7px;</string>
  </property>
  <widget class="QLabel" name="background_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>561</width>
     <height>346</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/images/images/id window11.png</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="enterButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>310</y>
     <width>80</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(0, 0, 0)</string>
   </property>
   <property name="text">
    <string>Sign in</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="passwordBox">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>270</y>
     <width>197</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QRadioButton" name="adminButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>115</y>
     <width>18</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="loginBox">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>230</y>
     <width>197</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QRadioButton" name="userButton">
   <property name="geometry">
    <rect>
     <x>234</x>
     <y>115</y>
     <width>18</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
